MVVMViews/ViewModel/SimpleSpellListViewModel.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.ObjectModel;
+using DnD5thEdTools.Models;
+using DnD5thEdTools.Repositories;
+using GalaSoft.MvvmLight;
+
+namespace MVVMViews.ViewModel
+{
+    /// <summary>
+    /// This class contains properties that a View can data bind to.
+    /// <para>
+    /// See http://www.galasoft.ch/mvvm
+    /// </para>
+    /// </summary>
+    public class SimpleSpellListViewModel : ViewModelBase
+    {
+        private readonly ISpellLoader _loader;
```

Comment:
I think the correct thing to use here is the ISpellListController which would provide more high level functionality. ISpellLoader is the lower level repository that just brings the data into the app.